src/02/03/z2ui5_cl_popup_to_select.clas.abap

```diff
@@ -81,7 +85,12 @@ CLASS z2ui5_cl_popup_to_select IMPLEMENTATION.
 
     DATA(columns) = tab->columns( ).
     LOOP AT lt_comp INTO ls_comp.
-      columns->column( width = '8rem' )->header( ns = `` )->text( text = ls_comp-name ).
+      DATA(text) = COND #(
+                     LET scrtext_m = CAST cl_abap_elemdescr( ls_comp-type )->get_ddic_field( )-scrtext_m IN
```

Comment:
as far as I remember, the get_ddic_field() does not work on steampunk/abap cloud programming model

Reply:
yes, maybe you can try to read it in the steampuncification way:
https://github.com/heliconialabs/steampunkification

Or just use the shortcut:
Make dynamic method calls out of it, then the check is also correct.

I know it is annoying, but i think its better to keep the project runnable on both language versions.

And the improvement is a nice idea, thank you for this!

Reply:
note: the method exists, it just shortdumps if I remember correctly

so maybe need to do https://github.com/heliconialabs/steampunkification?tab=readme-ov-file#find-abap-language-version-for-running-program before calling it

Reply:
Thanks for the hints. I'll try to come up with steampunk compatible solution. 
Of course it's important that this great project runs on all ABAP versions.

Reply:
Crafted a solution which runs (hopefully) on all ABAP versions.